client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Bluehost from './bluehost';
+import SiteGround from './siteground';
+import SectionHeader from 'components/section-header';
+
+class HostCredentialsPage extends Component {
+	constructor() {
+		super();
+		this.state = { fieldValues: {} };
```

Comment:
> We should use getInitialState instead of setting the initial state via this.state = ...

`this.state = ...` is also the standard way to set the initial state [when using ES6 style classes](https://facebook.github.io/react/docs/reusable-components.html#es6-classes). In 84dd990 I've removed it from the constructor and switched to using property initializers here too.

> We should probably use the redux store instead of state here

Personally I feel like Redux would be overkill here. This is a self-contained form where the state isn't needed elsewhere, and the state changes every time a character is typed into a field so we'd end up firing a Redux action on every keypress. We only really care about the contents of the form elsewhere in the app after the Submit button is pressed - at that point it'll get injected into a Redux action.

There's some interesting discussion on this topic in reactjs/redux#1287